assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -91,8 +91,8 @@ const doFailureBuffer = (
     setApiResponse(FAILURE_RESPONSE);
   } else {
     const elapsedMs = Date.now() - lastSuccess;
-
     if (elapsedMs < MINUTE_IN_MS) {
+      SentryLogger.info("Keeping current state.");
```

Comment:
I think when we get a request failure we should be logging the actual error, rather than this log which wouldn't tell us much other than "there was an error" (we know there are errors, the question is why). So in the places we _call_ `doFailureBuffer`, we should log the problem that resulted in us making that call. I see two instances of that in `useBaseApiResponse`:

1. If parsing the response fails. We could log the `response` in that case.
2. In the `catch` branch. We should capture the exception we caught.

Reply:
Ah okay, so are you suggesting we move the sentry logging up a level here to `fetchData()`? And should we keep the existing Sentry log for `Exiting no-data state`?

Reply:
Pretty much that, yes. And I think it's fine if we leave the existing log.